.github/workflows/create-release.yml

```diff
@@ -31,11 +31,11 @@ jobs:
               await github.rest.repos.createRelease({
                 draft: true,
                 generate_release_notes: true,
-                name: process.env.RELEASE_TAG,
+                name: import.meta.env.RELEASE_TAG,
                 owner: context.repo.owner,
                 prerelease: false,
                 repo: context.repo.repo,
-                tag_name: process.env.RELEASE_TAG,
+                tag_name: import.meta.env.RELEASE_TAG,
```

Comment:
to confirm VITE requires you to use `import.meta` ?

Reply:
Yep yep, [`import.meta` is how it's done in Vite](https://vitejs.dev/guide/env-and-mode#env-files). There might be a way to cast it to `process` instead if need be.

Reply:
I cast it to `process.env` so we could change less of the code.

Reply:
yeah, also just for note, this was a github workflow file, so it won't be included in the build.